Updated language (#47)
LibraryCarpentry/lc-marcedit: _episodes/01-introduction.md

### Manipulating MARC Data
MarcEdit is like the Swiss army knife for MARC data. You can perform single or global edits such as adding or deleting MARC fields, subfields, indicator values. Other edit functions are building fields using MARC data from other fields, or changing the case of data in a field. Other operations include running operations to help update your MARC data to the content standard Resource Description and Access (RDA) or to assign OCLC's FAST headings to records in the file.

### Tasks and Automation
Edits can be performed on one record by editing directly on the record or globally. If you have a list of changes to make, MarcEdit allows you to package those together to be run one after another. This is one of the ways to automate your work and makes manipulating your MARC data easier and quicker.

### Integrations
MarcEdit can integrate with OCLC's WorldCat and Ex Libris' Alma library system platform. Thanks to these integrations, you can retrieve records from external services, manipulate those records, and then push those changes back to the integration of your choice.

> Though not necessarily an integration, harvested records can also be manipulated. MarcEdit allows to harvest [OAI or Open Archives Initiative for Harvesting Metadata Protocole](https://www.openarchives.org/pmh/) Duplin Core or MARC21 records.

Reviewer: ```suggestion
Though not necessarily an integration, harvested records can also be manipulated. Using MarcEdit you can harvest [OAI or Open Archives Initiative for Harvesting Metadata Protocol](https://www.openarchives.org/pmh/) Dublin Core or MARC21 records.
```
